src/api/types.js

```diff
@@ -9,7 +9,11 @@
  */
 
 /**
- * @import { FormByIdInput, FormBySlugInput, FormDefinition, FormMetadataInput } from '@defra/forms-model'
+ * @typedef {Partial<FormMetadataDocument & { 'draft.updatedAt': Date, 'draft.updatedBy': FormMetadataAuthor }>} PartialFormMetadataDocument
```

Comment:
I'm not particularly keen on this but couldn't figure out a proportionate alternative. Open to suggestions else I'll leave it.

Reply:
For clarity, the reason this is required is because if you update the `{ draft: { ... }` it will update the entire nested tree for `draft` (i.e. it will drop `createdAt` and `createdBy`). Dot notation will preserve them.